src/applications/vaos/tests/new-appointment/components/TypeOfCarePage.unit.spec.jsx

```diff
@@ -191,6 +210,40 @@ describe('VAOS <TypeOfCarePage>', () => {
     );
   });
 
+  it('should open facility type page when CC eligible, has a support parent site, and v2 VAOS service requests are on', async () => {
+    const parentSite983 = {
+      id: '983',
+      attributes: {
+        ...getVAOSParentSiteMock().attributes,
+      },
+    };
+    const parentSite983GC = {
+      id: '983GC',
+      attributes: {
+        ...getVAOSParentSiteMock().attributes,
+      },
+    };
+    mockVAOSParentSites(['983'], [parentSite983, parentSite983GC]);
+    mockCommunityCareEligibility({
+      parentSites: ['983', '983GC'],
+      supportedSites: ['983GC'],
+      careType: 'PrimaryCare',
+    });
+    const store = createTestStore(initialStateVAOSService);
+    const screen = renderWithStoreAndRouter(
+      <Route component={TypeOfCarePage} />,
+      { store },
+    );
+
+    fireEvent.click(await screen.findByLabelText(/primary care/i));
+    fireEvent.click(screen.getByText(/Continue/));
+    await waitFor(() =>
+      expect(screen.history.push.lastCall?.args[0]).to.equal(
+        '/new-appointment/choose-facility-type',
+      ),
+    );
+  });
+
```

Comment:
What do you think about the following change?

```
    const store = createTestStore({
      ...initialState,
      featureToggles: {
        ...initialState.featureToggles,
        vaOnlineSchedulingVAOSServiceRequests: true,
      },
    });
```
...
```
    expect(global.fetch.firstCall.args[0]).to.equal(
      'https://dev-api.va.gov/vaos/v2/facilities?ids=983&children=false',
    );
```
This, to me, seems clearer that we are testing that the V2 endpoint is called. Also, what about, 'should open facility type page when CC eligible and has a support parent site'? This test is essentially the same as a previous test with the addition of turning of the vaOnlineSchedulingVAOSServiceRequests feature and the check for the correct URL.

Reply:
👍 I think those are good changes. I'll add those. 

> Also, what about, 'should open facility type page when CC eligible and has a support parent site'? 

Are you referring to changing the test name? `should open facility type page when CC eligible and has a support parent site` is already the name of the previous test. My current name of `should open facility type page when CC eligible, has a support parent site, and v2 VAOS service requests are on` is a little verbose so happy to change it if you have a suggestion.

Reply:
@vbahinwillit I removed our test and updated the test that @jbalboni wrote in another PR to use our parent facilities endpoint helper. It's under the `using VAOS service` describe block.